docs/content_management/pages/page_block_attributes.md

```diff
@@ -50,9 +50,7 @@ attributes:
 ```
 
 `embed` and `locationlist` use the Universal Discovery Widget (UDW).
-When creating a block with these two types you can configure the UDW behavior.
-
-For more information, see [Create custom page block / Add edit template](create_custom_page_block.md#add-edit-template)
+When creating a block with these two types you can use the `udw_config_name` option to configure the UDW behavior.
```

Comment:
I'm not sure about removing this part:
```
For more information, see [Create custom page block / Add edit template](create_custom_page_block.md#add-edit-template)
```

Is `udw_config_name`enough or are there still any "special cases" where you need to overwrite the template?

Reply:
`udw_config_name` is sufficient, ie:
```
        banner:
            attributes:
                contentId:
                    options:
                        udw_config_name: udw_banner_config
```

> I'm not sure about removing this part:
> (....)

I think it can remain. Why remove it?

Btw, there is also a embedvideo attribute type which seems to not be mentioned in documentation. `udw_config_name` works for that attribute type too